4.5 release page (#1138)
godotengine/godot-website: collections/_release_4_5/entry-platforms-android-editor-ui-adapted-for-non-desktop-users.md

---
type: entry
section: platforms
subsection: android
rank: 0
importance: 2
anchor: editor-ui-adapted-for-non-desktop-users
title: Editor UI adapted for non-desktop users
text: |

>   Users using a virtual keybord as their input method can attest that the UI of the editor is really desktop-oriented. Common actions – such as undoing and saving – are tedious to execute as they require each time to open up the toolbar and find the action.

Reviewer: ```suggestion
  Users using a virtual keyboard as their input method can attest that the UI of the editor is really desktop-oriented. Common actions – such as undoing and saving – are tedious to execute as they require opening up the toolbar each time to find the action.
```

>   Users using a virtual keyboard as their input method can attest that the UI of the editor is really desktop-oriented. Common actions – such as undoing and saving – are tedious to execute as they require opening up the toolbar each time to find the action.

Reviewer 2: ```suggestion
  Virtual keyboard users can attest to the desktop-oriented nature of the editor UI. Common actions – such as undoing and saving – are tedious to execute, as they require opening up the menu bar each time to find the action.
```
Would "menu bar" here in place of "toolbar" be correct for Godot?

>   With the new `TouchActionsPanel`, tablet users now have directly access to common actions buttons.

Reviewer: ```suggestion
  With the new `TouchActionsPanel`, tablet users now have direct access to common action buttons.
```

>   We intend to port this Android-only feature to other platforms as soon as possible.

Reviewer 2: ```suggestion
  This feature is currently Android-only, but we intend to port it to other platforms as soon as possible.
```
Emphasizing that the "Android-only" status is temporary
